playground/samples/simple.js

```diff
@@ -22,6 +22,19 @@ module.exports = {
         type: "string",
         title: "Bio",
       },
+      destination: {
+        type: "string",
+        title: "Where are you headed?",
+        enum: [
+          "GB",
+          "AU",
+          "France"
+        ],
+        enumNames: [
+          "United Kindom",
+          "Australia"
```

Comment:
I believe enumNames should have 3 elements here to correspond with the enum values.